rpcs3/Emu/RSX/gcm_enums.h

```diff
@@ -261,7 +328,7 @@ namespace gcm
 		CELL_GCM_CW = 0x0900,
 		CELL_GCM_CCW = 0x0901,
 
-		CELL_GCM_INVERT = 0x150A,
+		CELL_GCM_INVERT = 0x150A, // TODO: why is this a stencil op and not a logic op?
```

Comment:
And this.

Reply:
Because stencil and logic ops are just bitwise math. They just reused the same name for both since xor is xor no matter the context.

Reply:
For more context:
https://registry.khronos.org/OpenGL-Refpages/gl4/html/glLogicOp.xhtml
https://registry.khronos.org/OpenGL-Refpages/gl4/html/glStencilOp.xhtml

GL_INVERT is used for both raster operations. StencilOp is like LogicOp for stencil buffers.